eldev.el

```diff
@@ -111,6 +111,9 @@ instead.")
   "Name of Eldev cache subdirectory, `.eldev'.
 See also function `eldev-cache-dir'.")
 
+(defconst eldev-global-cache-directory-name "global-cache"
```

Comment:
Just use `-dir` instead of `-directory-name`, for consistency with e.g. `eldev-cache-dir`. I don't like to abbreviate too much, but sometimes names simply become too long otherwise.